requirements-common.txt

```diff
@@ -30,3 +30,7 @@ sdnotify==0.3.2
 
 # Api server
 flask==1.1.1
+
+# Support for colorized terminal output
+termcolor==1.1.0
+colorama==0.4.1
```

Comment:
i wouldn't necessarily depend on both ... (that's not what i meant, anyway).

are there benefits to do it this way?

Reply:
`init(autoreset=True)` would be useful if I did not use `colored` from termcolor, which handles all the reset sequences for us...

Well, as I said, these are just VT100 (ANSI) sequences, nothing more, nothing less (colorama also translates them to Windows sequences, so it's really needed). I found colorama offering exactly same approach -- a constant for a sequence, I would make same concatenations of the strings, probably implement those `red()`, `green()` functions again, but now with colorama constants instead of hardcoded... I found the termcolor `colored` interface convenient so it's been used...

It's not just arbitrarily, if you worry about this. There can be different targets during implementation, one of which is the code convenience for devs. Although this adds an extra dependency in this case...